src/crimson/os/seastore/random_block_manager/rbm_device.h

```diff
@@ -165,12 +173,28 @@ class RBMDevice : public Device {
   virtual std::string get_device_path() const = 0;
 
   uint64_t get_journal_size() const {
-    return super.journal_size;
+    return shard_info.journal_size;
   }
 
-  static rbm_abs_addr get_journal_start() {
+  static rbm_abs_addr get_journal_start_off() {
     return RBM_SUPERBLOCK_SIZE;
   }
+
+  rbm_abs_addr get_journal_start() {
+    return shard_info.start_offset + get_journal_start_off();
+  }
+
+  uint64_t get_offset() const {
+    return shard_info.start_offset;
+  }
+
+  uint64_t get_end() const {
```

Comment:
They are shard-local interfaces.

Perhaps the better names would be:
* get_shard_journal_start()
* get_shard_data_start()
* get_shard_end()